src/modules/UploadModule.ts

```diff
@@ -119,6 +115,20 @@ export default class UploadModule extends VuexModule implements FfaProcessModule
     return FileHelper.fileSizeString(this.file.size)
   }
 
+  get hash(): string {
+    const metaMaskModule = this.context.rootState.metaMaskModule as MetaMaskModule
+    if (metaMaskModule.publicWalletAddress.length === 0) {
+      throw new Error(Errors.ADDRESS_EMPTY)
+    }
+    if (this.title.length === 0) {
+      throw new Error(Errors.TITLE_EMPTY)
+    }
+    const hashedAccount = keccak256(metaMaskModule.publicWalletAddress)
+    const hashedTitle = keccak256(this.title)
+    const hash = keccak256(`${hashedAccount}${hashedTitle}`)
+    return `0x${hash}`
```

Comment:
make sure 0x isn't already there. i remember i changed hashing libs once and ended up with `0x0x...`